packages/trader/src/Modules/CFD/Containers/cfd-dashboard.jsx

```diff
@@ -43,7 +43,7 @@ const CFDDashboard = props => {
     const [is_demo_enabled, setIsDemoEnabled] = React.useState(false);
     const [is_real_enabled, setIsRealEnabled] = React.useState(false);
     const [active_index, setActiveIndex] = React.useState(0);
-    const [is_demo_tab, setIsDemoTab] = React.useState(true);
+    const [is_demo_tab, setIsDemoTab] = React.useState(false);
```

Comment:
on the first load of the page, the real tab is always displayed, which means that the value of the displayed tab must match